File: PHP/Example Array/Looping Data (Array).php
<!DOCTYPE HTML>
<html>

<head>
	<title>Example Looping Data Array</title>
</head>

<body>

	<?php
	$varlttr = array("a", "b", "c", "d", "e", "f", "g", "h", "i", "j");
	for ($i = 0; $i < 10; $i++) {
		echo "Letter : " . $varlttr[$i] . "<br>";
	}
	?>

</body>

</html>

File: C#/Example Array/Looping Data  (Array).cs
using System;

namespace Main
{
    class main
    {
        static void Main(string[] args)
        {
            char[] varlttr = { 'a', 'b', 'c', 'd', 'e', 'f', 'g', 'h', 'i', 'j' };

            for (int i = 0; i < 10; i++)
            {
                Console.WriteLine($"Letter : {varlttr[i]}");
            }
        }
    }
}
